apps/omg_watcher_rpc/lib/tracer.ex

```diff
@@ -18,4 +18,69 @@ defmodule OMG.WatcherRPC.Tracer do
   """
 
   use Spandex.Tracer, otp_app: :omg_watcher_rpc
+
+  @doc """
+  Given conn and parsed response body, return a span with extra metadata.
+
+  The service medata (default ':web') is overridden to be the name of the OMG service.
+
+  Metadata added to the span is not arbitrary and is validated by Spandex against a schema:
+
+  service(:atom) Required: The default service name to use for spans declared without a service
+  env(:string): A name used to identify the environment name, e.g prod or development
+  services([{:keyword, :atom}, :keyword]): A mapping of service name to the default span types. - Default: []
+  completion_time(:integer)
+  error(:keyword)
+  http(:keyword)
+  id(:any)
+  name(:string)
+  parent_id(:any)
+  private(:keyword) - Default: []
+  resource([:atom, :string])
+  sql_query(:keyword)
+  start(:integer)
+  tags(:keyword) - Default: []
+  trace_id(:any)
+  type(:atom)
+  """
+  defp add_metadata_from_response_body(conn, json_resp_body) do
+    service = String.to_atom(json_resp_body["service_name"])
```

Comment:
this seems somewhat dangerous that could peraps blow up the VM unintentionally. try to choose the atom, for example:
```
case (json_resp_body["service_name"] do
"watcher"->:watcher
"watcher_info"-> :watcher_info
end
```
the reason being, atoms are not garbage collected and turning strings to atoms is something we should avoid.